pennylane/templates/subroutines/qpe.py

```diff
@@ -200,6 +200,30 @@ def estimation_wires(self):
         """The estimation wires of the QPE"""
         return self._hyperparameters["estimation_wires"]
 
+    @property
+    def error(self):
+        """The QPE error computed from the spectral norm error of the input unitary operator.
+
+        **Example**
+
+        >>> class CustomOP(qml.resource.ErrorOperation):
+        ...    @property
+        ...    def error(self):
+        ...       return 0.1
+        >>> Op = CustomOP(wires=[0])
+        >>> QPE = QuantumPhaseEstimation(Op, estimation_wires = range(1, 5))
+        >>> QPE.error
+        0.11010001000000001
+        """
```

Comment:
So for my understanding, this is the operator error for QPE found by composing the errors of the `Op` unitary being applied? How is this related to the error one would get from QPE, i.e., the error in measuring the phase (which depends on things like the number of qubits in the phase estimation register)?

Reply:
> So for my understanding, this is the operator error for QPE found by composing the errors of the Op unitary being applied?

Yes. 

> How is this related to the error one would get from QPE, i.e., the error in measuring the phase (which depends on things like the number of qubits in the phase estimation register)?

This will be addressed in future extension that address the estimation error. This [reference](https://arxiv.org/abs/2111.10430) provides some insights on this.